src/lc3_vm/flags.rs

```diff
@@ -0,0 +1,16 @@
+#[allow(clippy::upper_case_acronyms)]
+enum ConditionFlags {
+    POS = 0,
+    ZRO = 2,
+    NEG = 4,
```

Comment:
Is the value here necessary? You translate the values to u16 in the `From<ConditionFlags>` function; and they are the same values. 

Would there be a problem if these values get out-of-sync?